### ISSUE TITLE
cannot import name 'Property' at jumpscale.data.bcdb

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/js-sdk#102]
after calling `threebot start`

```
~> threebot start   
Traceback (most recent call last):
  File "/home/ahmed/.cache/pypoetry/virtualenvs/js-sdk--OGPzDz5-py3.6/src/js-ng/jumpscale/god.py", line 156, in loadjsmodules
    m = importlib.import_module(importedpkgstr)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ahmed/wspace/js/js-sdk/jumpscale/data/bcdb/__init__.py", line 1, in <module>
    from .bcdb import *
  File "/home/ahmed/wspace/js/js-sdk/jumpscale/data/bcdb/bcdb.py", line 3, in <module>
    from jumpscale.data.bcdb import models as models
  File "/home/ahmed/wspace/js/js-sdk/jumpscale/data/bcdb/models/__init__.py", line 1, in <module>
    from .model_model import ModelBase
  File "/home/ahmed/wspace/js/js-sdk/jumpscale/data/bcdb/models/model_model.py", line 3, in <module>
    from .base import ModelBase, JSObjBase
  File "/home/ahmed/wspace/js/js-sdk/jumpscale/data/bcdb/models/base.py", line 2, in <module>
    from jumpscale.data.schema import Property
ImportError: cannot import name 'Property'
[-] cannot import name 'Property' at jumpscale.data.bcdb 

1. mainnet
2. testnet
3. devnet
4. none
Which network would you like to register to? 


***

**Comment by xmonader** [comment]
should be fixed at c8012a8a52295cd8a6690cd5af19763ff642322d

**Comment by hossnys** [comment]
verified 

![Screenshot from 2020-06-28 10-46-46](https://user-images.githubusercontent.com/26298009/85942839-407f3700-b92c-11ea-9181-e26495e5f24e.png)
